### PR TITLE
Version 2.1.1

### PR DESCRIPTION
**Pull request by plankter**
- Update `xtiff` package
- FIX: Wrong ROI coordinates for new Fluidigm software versions #104
- FIX: Extra 161 bytes extracted on all image buffers #102

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ packaging = "*"
 pandas = "^1.0"
 typing_extensions = ">=3.7.4.2"
 xmltodict = ">=0.12.0"
-xtiff = ">=0.6.2"
+xtiff = ">=0.6.3"
```
Comment (jwindhager) [inline pyproject.toml:32]: This should be 0.6.4

File: imctools/converters/ome2analysis.py [hunk 2]
```diff
@@ -175,26 +175,26 @@ def omefolder_to_analysisfolder(
...
+    #         "/home/anton/Downloads/imc_folder/20170905_Fluidigmworkshopfinal_SEAJa/20170905_Fluidigmworkshopfinal_SEAJa_s0_a0_ac.ome.tiff"
+    #     ),
+    #     Path("/home/anton/Downloads/analysis_folder"),
+    #     "test",
+    #     panel_csv_file=Path("/home/anton/Downloads/example_panel.csv"),
     #     metalcolumn="Metal Tag",
+    #     usedcolumn="ilastik",
     # )
 
+    omefolder_to_analysisfolder(
```
Comment (jwindhager) [inline imctools/converters/ome2analysis.py:189]: Hard-coded file paths to Anton's user directory

**Review by jwindhager — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (plankter) [reply pyproject.toml:32]: It is still 0.6.3 on PyPI: https://pypi.org/project/xtiff/

File: imctools/converters/ome2analysis.py (hunk 2 above)
Comment (plankter) [reply imctools/converters/ome2analysis.py:189]: Yes, I am aware of it. These are used for debugging only, as it was with v1.

**Pull request merged**